doc/sphinx/source/inspection/index.rst

```diff
@@ -9,9 +9,9 @@ is generated, along with a ``png``. The default can be adjusted by setting the
 ``NGRAPH_VISUALIZE_TREE_OUTPUT_FORMAT`` flag to another format, like PDF. It
 can then be inspected with a third-party visualization tool such as:
 
-#. `Netron`_
+.. #. `Netron`_ support tentatively planned to come soon
 
-#. `TFBoard`_
+#. `TensorBoard`_
```

Comment:
I think we should remove this reference altogether. It cannot be used to visualize the graphs you speak about in the preceding paragraph.

Reply:
I see the word "cannot" there now.  Thanks.

Reply:
Agree